src/nvim/ui.c

```diff
@@ -53,6 +53,10 @@ static bool pending_cursor_update = false;
 static int busy = 0;
 static int height, width;
 
+static bool tabline_external = false;
+static bool cmdline_external = false;
+static bool wildmenu_external = false;
```

Comment:
I'm confused about why this extra state is needed. Can the `uis` be iterated to find the flag instead?